.github/workflows/create-cli-release.yml

```diff
@@ -93,9 +93,27 @@ jobs:
       channel: ${{ needs.get-channel.outputs.s3-channel }}
     secrets: inherit
 
-  # TODO: Add docker-build-full and docker-build-slim here
-  #       Also add them to the announce-cli-patch-in-slack job "needs"
+  build-docker-slim:
+    # TODO: remove when GA
+    if: ${{ !contains(fromJSON('["latest", "latest-rc", "nightly"]'), needs.get-channel.outputs.channel) }}
```

Comment:
Do we need these checks in `v2`? Especially since we have changed the image location to `salesforce/cli`?